bdb/bdb_api.h

```diff
@@ -546,8 +546,11 @@ int bdb_close_temp_state(bdb_state_type *bdb_state, int *bdberr);
 
 /* get file sizes for indexes and data files */
 uint64_t bdb_index_size(bdb_state_type *bdb_state, int ixnum);
```

Comment:
Maybe we could get rid of the non-tran version (i.e., `bdb_index_size`, `bdb_data_size` and `bdb_queue_size`). They're not used anywhere in your patch. Doing so also makes it clear that these functions should be invoked transactionally.

Reply:
YES .. lemme rework this a bit .. thanks Rivers!